### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: translations/locale_de.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    </message>
+</context>
+<context>
+    <name>QMessageBox</name>
+    <message>
+        <location line="-221"/>
+        <source>No score open.
+This plugin requires an open score to run.
+</source>
+        <translation>Keine Partitur geöffnen.
```
Comment (RobFog) [inline translations/locale_de.ts]: - geöffnet :-)

Comment (Jojo-Schmitz) [inline translations/locale_de.ts]: Ahem.... will fix shortly, thanks ;-)

**Comment by jeetee** [comment]
Thanks for the effort. Would you mind not including the translations that are not in the TempoChanges context?

I probably should clarify that with a readme in the translations folder, but all other contexts are already included within MuseScore, so we can leverage those translations directly.

**Comment by Jojo-Schmitz** [comment]
done

**Pull request merged**